api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCommon.java

```diff
@@ -161,19 +163,84 @@ public static File getTFLiteAddModel() {
      * CPU: "custom=ModelFWType:CAFFE,ExecutionDataType:FLOAT32,ComputingUnit:CPU"
      * GPU: "custom=ModelFWType:CAFFE,ExecutionDataType:FLOAT32,ComputingUnit:GPU,GpuCacheSource:/sdcard/nnstreamer/"
      */
-    public static String getSNAPCaffeOption(boolean useGPU) {
+    public static String getSNAPCaffeOption(SNAPComputingUnit CUnit) {
         String option = "ModelFWType:CAFFE,ExecutionDataType:FLOAT32,InputFormat:NHWC,OutputFormat:NCHW,";
 
-        if (useGPU) {
-            String root = Environment.getExternalStorageDirectory().getAbsolutePath();
-            option = option + "ComputingUnit:GPU,GpuCacheSource:" + root + "/nnstreamer/";
-        } else {
-            option = option + "ComputingUnit:CPU";
+        switch (CUnit) {
+            case CPU:
+                option = option + "ComputingUnit:CPU";
+            case GPU:
+                String root = Environment.getExternalStorageDirectory().getAbsolutePath();
+                option = option + "ComputingUnit:GPU,GpuCacheSource:" + root + "/nnstreamer/";
+            case DSP:
+                break;
```

Comment:
You can omit break here

Reply:
I removed this switch code. The option string is now from enum's getter function.